pcap/stream.go

```diff
@@ -314,6 +359,26 @@ func (c *tcpStream) Accept(tcp *layers.TCP, _ gopacket.CaptureInfo, dir reassemb
 		}
 	}
 
+	// One of the flows initiated a connection close request
+	if tcp.FIN {
+		if !revFlow.FirstPacketSeen() {
+			// The current flow is the one that initiated the connection close request
+			// and no packets have yet arrived on the reverse flow
+			// this would indicate a timeout on the current side
```

Comment:
I really thing we need to do the design work here of showing the different scenarios and listing out the possibilities to be sure we're covering these correctly.

Client shutdown:
```
client -> SYN
               SYN-ACK <- server
client -> SYN
client -> HTTP request  
               ACK   <- server
              ...
client -> FIN
               FIN-ACK <- server
```

Are we called on every TCP frame, or only those with payloads?  If every TCP frame, then wouldn't revFlow.FirstPacketSeen() be true?  We probably need to set it only when there are bytes in the stream.

Server shutdown:
```
as above until
               ...
               FIN <- server 
client -> FIN-ACK
```

It seems to me that currFlow.TcpFlowInitiator() is not set, because we have not selected a parser.

These are subtle enough that we need unit tests for this code.